build/FrameworkPackage/FrameworkPackage.csproj

```diff
@@ -0,0 +1,218 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="ProjectConfigurations">
```

Comment:
we can probably delete this file. I think for a while I was trying to get the FrameworkPackage to be built in VS (and I still think it's possible) but for the workflow it isn't important so I never finished it.

Reply:
ok